src/main/kotlin/race/SimpleCar.kt

```diff
@@ -1,9 +1,23 @@
 package race
 
-class SimpleCar(private var location: String = "") {
-    fun getCurrentlocation() = location
+import java.lang.IllegalArgumentException
 
-    fun move() {
-        if ((0..9).random() > 3) location += "-"
+class SimpleCar(val name: String) {
+    private var location = 0
+
+    init {
+        if (name.length > 5) {
+            throw IllegalArgumentException("자동차 이름은 5자를 초과할 수 없습니다")
+        }
+    }
+
+    fun getLocation() = location
```

Comment:
조금 더 코틀린 스럽게 표현을 해볼까요?

```suggestion
    val localtion: Int
        get() = _location // 기존 location을 _location으로 변경한다는 가정
```